src/types.rs

```diff
@@ -28,6 +28,21 @@ pub trait Interface: Send + Sync + InterfaceClone {
         bail!("unimplemented function get_module in interface")
     }
 
+    /// Transfer an amount from the address on the current call stack to a target address.
+    fn transfer_coins(&self, _to_address: &Address, _raw_amount: u64) -> Result<()> {
+        bail!("unimplemented function transfer_coins in interface")
```

Comment:
let's add it to the interface as well

Reply:
This should be implemented in `massa`, as done in https://github.com/massalabs/massa/pull/2059

Reply:
True :) I'll do it